Amazon.QLDB.Driver.IntegrationTests/StatementExecutionTests.cs

```diff
@@ -830,17 +828,25 @@ public void Execute_ExecutionMetrics()
             qldbDriver.Execute(txn =>
             {
                 var result = txn.Execute(selectQuery);
+                Driver.IOUsage? ioUsage = null;
```

Comment:
We don't need to create ioUsage and timingInfo variables here.

Stream cursor query stats will accumulate the stats itself. we just need to set `readIO = ioUsage.Value.ReadIOs`.

Reply:
Addressed